app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingPrivacyNoticeViewHolder.kt

```diff
@@ -19,10 +19,10 @@ class OnboardingPrivacyNoticeViewHolder(
 
     init {
         val binding = OnboardingPrivacyNoticeBinding.bind(view)
-        binding.headerText.setOnboardingIcon(R.drawable.ic_info)
 
-        val appName = view.context.getString(R.string.app_name)
-        binding.descriptionText.text = view.context.getString(R.string.onboarding_privacy_notice_description2, appName)
+        binding.headerText.text = view.context.getString(R.string.onboarding_privacy_notice_header_1)
+
+        binding.descriptionText.text = view.context.getString(R.string.onboarding_privacy_notice_description)
```

Comment:
We don't need to set the text in the ViewHolder since we already set the text in the xml. We set the text prior because we need to manually compute the `appName`.